src/app/shared/dovetail/dovetail-audio.ts

```diff
@@ -73,6 +73,8 @@ export class DovetailAudio extends ExtendableAudio {
     this.$$sendEvent(PLAY);
     if (this._dovetailLoading) {
       this.resumeOnLoad = true;
+    } else if (this.arrangement.entries[this.index].unplayable) {
+      this.skipToFile(this.index + 1, true);
```

Comment:
If the 1st file in the arrangement is unplayable, it has already been loaded (and the playback error caught) by the time we get here.  I think it's trying to get the duration of the 1st segment before you press play.

So without this `else if` ... the `_audio.play()` doesn't do anything.